internal/controller/manifests_downloader.go

```diff
@@ -38,9 +38,6 @@ import (
 )
 
 const (
-	configMapVersionLabel     = "provider.cluster.x-k8s.io/version"
```

Comment:
what is the benefit of moving these to API package? AFAICT, they are used in internal package only

Reply:
Yes, this is good question, but according to test data below, it seems to possible to have any ConfigMap name with following labels to be configured in air-gap mode, to avoid have name equals to version.

This would be an option for configuration. Seems to need to add this option to documentation.
What do you think about it?

https://github.com/kubernetes-sigs/cluster-api-operator/blob/6b72fcd6c67e60a1f80ef1f367f065d4adec7019/test/e2e/resources/core-cluster-api-v1.7.7.yaml#L13358-L13360